deploy/resources/clusterrolebindings.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: federation-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: federation-role
+subjects:
+- kind: ServiceAccount
+  name: federation-controller
+  namespace: '{{ .Release.Namespace }}'
```

Comment:
Strike this line and add a transform in the controller's code that sets the correct namespace